src/js/setUpSite.ts

```diff
@@ -150,7 +150,17 @@ const snapToCity = (map: Map, layer: ImageOverlay): void => {
   map.fitBounds(layer.getBounds());
 };
 
-const setScorecard = (cityId: CityId, cityProperties: ScoreCard): void => {
+/**
+ * Sets scorecard to city and alert if city is community made.
+ *
+ * @param cityId: E.g. `columbus-oh`.
+ * @param cityProperties: An object with a `layout` key (Leaflet value) and keys
+ *    representing the score card properties stored in `score-cards.json`.
+ */
```

Comment:
The types make these comments less necessary

```suggestion
/**
 * Sets scorecard to city and alert if city is community made.
 */
```